test.cfg.example

```diff
@@ -0,0 +1,4 @@
+user=*****
+password=*****
+# the path to the MongoDB executable.
+test-mongod-exe=/kb/runtime/bin/mongod
```

Comment:
Actually, AFAICT no tests use a temp.dir parameter - the temp dir is always hard coded to ./temp_files